### ISSUE TITLE
Add support for SLE Micro

### ISSUE THREAD

**Issue by davidnuzik** [k3s-io/k3s#3188]
QA test, see if can identify any issues. Does SELinux work when enforcing?
Test with SLE Micro 5.0

**Comment by davidnuzik** [comment]
This is to remain in the backlog for now. Perhaps later we'll want to QA test and certify SLE Micro 5.0, 5.1 or later version down the road.

**Comment by rancher-max** [comment]
This has been mostly validated on v1.22.3+k3s1, v1.21.6+k3s1, and v1.20.12+k3s1 following our New OS Support Matrix (installing rancher as local cluster, running sonobuoy conformance checks, using docker cri, and upgrades via SUC). I have not run some automated kubernetes checks through rancher yet, which is the final validation we like to do.

One thing I'd like to call out is that in all of these cases, I did this through VirtualBox VMs, which was taxing on my machine so I was only able to run a single server and single agent (although did see another server able to join in one case before I ran OOM). For the final check on all versions, I will upload the SLE Micro ISO to an internal server and use that to install it on 3 servers, 1 agent, and perform the final check with that configuration.

This is all done with SLE Micro 5.1
